docs/using/backup.md

```diff
@@ -0,0 +1,93 @@
+# Backup and Restore
+
+> <b style="color:goldenrod">Caution</b> Backup and restore features are a technical preview and currently only work with MySQL database servers backing up to Amazon AWS S3 storage locations.
+
+## Before you start
+
+- You have an AWS S3 storage account and location details for it.
+- There is only one MySQL instance running on the node.
+- MySQL is running:
+    - as a service via `systemd`;
+    - with the name `mysql` (to confirm, use `systemctl status mysql`);
+    - from a `mysql` system user account.
+- There is a `mysql` system group.
+- MySQL is using the `/var/lib/mysql` directory for database storage.
+- Backup management has been enabled:
+    1. Navigate to *{{icon.cog}} Configuration-->Settings-->Advanced Settings*.
+    2. Activate *Backup Management*.
+    3. Click *Apply changes*.
+    4. In the left menu bar, click *{{icon.history}}-->Backup*.
+
+        ![!](../_images/PMM_Backup_Management.jpg)
+
+- The following packages are needed for creating backups. They should be included in the `$PATH` environment variable:
+    - [xtrabackup][PERCONA_XTRABACKUP], which includes:
+        - [`xbcloud`][PERCONA_XBCLOUD]
+        - [`xbstream`][PERCONA_XBSTREAM]
+    - [`qpress`][PERCONA_QPRESS]
+
+    **The versions of each must be compatible with the installed version of MySQL.**
+
+
+## Adding a storage location {: #backup-location }
+
+1. Navigate to Backup Management-->Storage locations
+2. Click *Add*
+3. Fill in the form fields
+
+    ![!](../_images/PMM_Backup_Management_Locations_Add_Storage_Location.jpg)
+
+    - *Name*:
+    - *Description*:
+    - *Type:*
+        - *S3*:
+        - *Local Client:*
+        - *Local Server:*
+    - *Endpoint:*
+    - *Bucket Name:*
+    - *Access Key:*
+    - *Secret Key:*
+
+4. Click *Add* to add the location or *Test* to test the connection.
+
+## Creating a backup {: #backup-create }
+
+> You must add a [backup storage location](#backup-location) before backing up
+
+1. Navigate to *Backup Management-->Backup Inventory*
+
+2. Click {{icon.plussquare}} Add
+
+3. In the *Backup On Demand* dialog, enter values for:
+
+    - *Service name* -- The service being backed up
+    - *Vendor* -- The database vendor name (only * * currently supported)
```

Comment:
vendor is read only. User don't have to enter values. And for now only mysql is supported.